### PR TITLE
Use init without raven if DOM isn't loading

### PR DESCRIPTION
**Pull request by davidpmccormick**
## What is this PR trying to achieve?
Firing the JS at the right time.

### CODE REVIEW

File: client/js/app.js [hunk 1]
```diff
@@ -73,7 +73,7 @@ function initWithRaven() {
 // If the DOM is not loading, we can init, else wait till we do
 const domNotLoading = document.readyState !== 'loading';
 if (domNotLoading) {
-  initWithRaven();
+  init();
```
Comment (davidpmccormick) [inline client/js/app.js]: This is what the comment says it should do – presume this was a typo?

Comment (jamesgorrie) [inline client/js/app.js]: Hmm. I don't like this as then we don\t get error handling.
How can we make sure raven is available?

Comment (jamesgorrie) [inline client/js/app.js]: I think we might be able to npm install raven and then include it?

**Comment by davidpmccormick** [comment]
Closing as a red herring.

**Pull request closed without merging**